infra/modules/database/README.md

```diff
@@ -0,0 +1,25 @@
+# Java multizone database module
```

Comment:
Since this originally an internal submodule (i.e any module in modules/) without a readme, we should skip metadata.